org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/utils/ConfigurationItemEditType.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.extensions.contentmodel.utils;
+
+/**
+ * Configuration item edit type. Represents a configuration edit operation
+ */
+public enum ConfigurationItemEditType {
+	add, delete
```

Comment:
The names of enums in the language server protocol generally have the first letter of their name capitalized, and the Java convention is usually to use all caps. I think it might be a good idea to use one of those conventions in order to be consistent.